operations/finance.md

```diff
@@ -669,13 +669,13 @@ Returns all outlet items of the enterprise that were consumed \(posted\) or will
 | `ClosedUtc` | string | required | Date and time of the bill closure in UTC timezone in ISO 8601 format. |
 | `Notes` | string | optional | Additional notes on the bill. |
 
-## Get all credit cards by ids
+## Get all credit cards 
 
-Returns all credit cards with the specified ids.
+Returns all credit cards filtered by at least one of these properties: [Customer](customers.md#customer) identifier, [Credit card](finance.md#credit-card) identifier
```

Comment:
Returns all credit cards, possibly filtered by Customers, identifiers and other filters.

Reply:
Missing dot at the end.